README.md

```diff
@@ -1,16 +1,12 @@
 # infrastructure
 
-Repo to track code + issues relating to IPFS infrastructure for users + devs.
-This includes:
+This repository contains the technical infrastructure of the IPFS community.
 
-Computing:
-- Gateway nodes
-- Bootstrap nodes
-- CI servers (jenkins, travis, etc)
+- Gateway (http://gateway.ipfs.io) and bootstrap nodes, in `solarnet/`
+- Seeding of $important objects, in `seeding/`
 
-Communications:
-- Github repos
-- IRC Channel
-- Mailing Lists
```

Comment:
My idea here was to avoid duplication and keep that in ipfs/community.git, so it'd be about "using the infrastructure", while ipfs/infrastructure.git would be about "running the infrastructure".

But maybe this distinction doesn't make sense? We should maybe look into merging the two then.